### DISCUSSION TITLE
Show the Queue mode of the room in the Multi Lounge

### DISCUSSION THREAD

**Discussion by minasminas** [ppy/osu#15956] [Ideas] [answered]
I think showing the queue mode directly in the Multi Lounge would help to decide which lobby to join (if you specificly want to join a round robin lobby for example).

**Comment by bdach** [comment, marked as answer]
Partially overlaps with https://github.com/ppy/osu/issues/15675

**Reply by peppy** [reply]
Has been resolved via a text display for now, via https://github.com/ppy/osu/pull/15976.
